Release/v1.0.3

W3stside opened the pull request:
`Provider` logic changes

Open to suggestions on what the Object looks like

Velenir committed:
METAMASK -> activeProvider

Velenir committed:
unlock check

W3stside committed:
changed updateProvider getting incorrect param

W3stside committed:
added local generated web3 to replace window web3 + integration fix

1. Mist works somewhat (loads and doesnt crash but still requires network change)
2. added local web3 to window web3 as only provider added by Mist

W3stside committed:
renamed intrations/metamask to providers

1. changed routing to work with new name

W3stside committed:
created walletintegration container

W3stside committed:
added ledger to providers file

W3stside committed:
added WalletIntegrations container to App

W3stside committed:
Webpack http localhost >> https localhost

W3stside committed:
only call localSettings load before app

W3stside committed:
transition promise wait to promise function wait + provider logic

1. if provider needs updating in updateMainAppState + activeProvider
not set, call for default via priority

W3stside committed:
API restructure

1. API init (dx, tokens, contracts etc) is a fn call
2. changed where necessary in code
3. logging to show api firing stages

W3stside committed:
fixed api call from awaiting var to fn call + types/stories

W3stside committed:
logic simplification in bc/selectors and added type to bc state

1. providers can save their types in state, remove activeProvider
setting from registerProvider

W3stside committed:
AppValidator logic update for new Providers

W3stside committed:
added network2rpcUrl/types/ledger in globals

W3stside committed:
WalletIntegration: logic update, Loader comp add, SCSS

W3stside committed:
providerWatcher logic update to work with providerType + bug fix

W3stside committed:
walletIntegration logic refactor + moved fns out into Providers

a. types edit
b. Providers has fns + some logic check for Ledger

W3stside committed:
adding in SVGs and style rules

W3stside committed:
adding gas and gasLimit to api functions

W3stside committed:
types and globals

W3stside committed:
saving provider.keyname

W3stside committed:
removed breaking stopFn from watcher w/Ledger

W3stside committed:
provider selector by keyName

W3stside committed:
added new subproider + watching

W3stside committed:
test fn in walletintegration to test ledger

W3stside committed:
force show ledger wallet

W3stside committed:
error fixes

W3stside committed:
fixed develop merge conflicts

W3stside committed:
AppValidator logic changes + props add

1. if wallet locked during load, dont reload page but update state
2. show loader using loading state prop until setup complete

W3stside committed:
WalletIntegrationProvider container logic changes

1. setActiveProviderSet local state var to test un-activiation of activeProvider
2. shows wallet screen when activeProvider gone

W3stside committed:
gnosis safe svg

W3stside committed:
added checks for gnosis safe in providers and walletintegration

W3stside committed:
Merge branch 'develop' of github.com:gnosis/dx-react into DX-517/LEDGER-Integration

W3stside committed:
added more provider names to types and provider init

W3stside committed:
function for grabbing correct provider svg based on name

W3stside committed:
walletIntegration container to use providerInfo2Svg function

W3stside committed:
p-lock and webpack dev back to http to test Status

W3stside committed:
changed styling to smaller icons

W3stside committed:
deals with better presentation of wallets on mobile

W3stside committed:
mass exported provider SVGs and fixed dynamic load

W3stside committed:
merged in DX-517/Ledger Integration essential code

W3stside committed:
removed dead code, fixed styles, p-lock

W3stside committed:
fixed overflow issue in cookies article in mobile

W3stside committed:
Loader takes strokeWidth + strokeColor props + defaults

W3stside committed:
fixed Loader color issue in AppValidator

W3stside committed:
If only 1 Provider, immediately start app, no walletChooser

W3stside committed:
fixed cookie mobile layout display

W3stside committed:
Fixes issue where in MOBILE layout is unreadable

1. postSellOrder || depositAndSell modal - 'Final Confirmation' was unscrollable
2. removes noScroll ONLY on this modal

W3stside committed:
fixes Cookies content text bug

W3stside committed:
added proper provider name since change

Velenir commented on webpack.dev.config.js:
```diff
@@ -125,6 +125,7 @@ module.exports = {
   devServer: {
     disableHostCheck: true,
     historyApiFallback: true,
+    https: false,
```
It's `false` by default. Was there a problem before?

Velenir commented on src/integrations/provider/index.ts:
```diff
@@ -1,29 +1,138 @@
+  INJECTED_WALLET: {
+    priority: 90,
+    providerType: 'INJECTED_WALLET',
+    keyName: 'INJECTED_WALLET',
```
What's `keyName` about?

W3stside committed:
removed HTTPS false as default

W3stside committed:
fixes issue if no wallets are active

W3stside committed:
WalletIntegration in Router + broken Disclaimer

1. not rerouting properly

Velenir committed:
Fix redirect to /disclaimer (#354)

* fix blocking updates

* extra ' } '

Velenir committed:
DX-568/fix children remount (#353)

* allow for easier element reconciliation

* Loader re-added

1. fixes problem of status showing up during load

W3stside committed:
imprint

W3stside committed:
Hamburger menus changed

Velenir committed:
Tokens content

Velenir committed:
add content images

Velenir committed:
add typings for *.png

Velenir committed:
add Fees content

Velenir committed:
style images in Fees content

Velenir committed:
fix AppValidator not updating on location change

Velenir committed:
fix AppValidator not updating on location change

W3stside committed:
Change of content

Velenir committed:
add FAQ contents

W3stside committed:
Merge pull request #355 from gnosis/feature/imprint-release

Release/1.0.3 --> Imprint

W3stside committed:
Merge pull request #356 from gnosis/release_1.0.3/Hamburger-Menus

Release/1.0.3 --> Hamburger menus

W3stside committed:
Merge pull request #357 from gnosis/release_1.0.3/How-it-Works

Release/1.0.3 --> How it Works

W3stside committed:
Release/1.0.3 --> Help & Contact (#358)

* removed comments and imprint

* changed Help/Contact page content page

1. passed in network prop via container to conditionally render Help page for RInkeby/Main

W3stside committed:
merged in 1.0.3 and fixed conflicts

W3stside committed:
Merge branch 'content_page_changes' into release/1.0.3

W3stside committed:
added missing link
